src/components/DetailsCard.js

```diff
@@ -0,0 +1,46 @@
+import { format } from 'd3-format'
+import React from 'react'
+
+const DetailsCard = ({
+  data,
+  title,
+}) => {
+  const colWidths = ['3', '2', '4', '3']
+  const formatPercent = format('.0%')
+  const formatNumber = format(',')
+  const sum = data.reduce((total, next) => (total + next.count), 0)
+  const withPercent = data.map(d => ({ ...d, percent: (d.count / sum) }))
```

Comment:
@brendansudol This PR is great! I just made this small change so that the component takes less data through the props.